model/textparse/nhcbparse_test.go

```diff
@@ -509,7 +509,6 @@ something_bucket{a="b",le="+Inf"} 9 # {id="something-test"} 2e100 123.000
 				{Labels: labels.FromStrings("id", "something-test"), Value: 0.0, HasTs: true, Ts: 123321},
 				{Labels: labels.FromStrings("id", "something-test"), Value: 2e100, HasTs: true, Ts: 123000},
 			},
-			// TODO(krajorama): ct:   int64p(1520430002000),
```

Comment:
note to reviewers: input doesn't have created timestamp, copy paste error to have this comment.